processes/presenting-at-conferences.rst

```diff
@@ -0,0 +1,154 @@
+#########################
+Presenting at Conferences
+#########################
+
+This is our playbook for presenting on behalf of DM at both project and public meetings.
+
+.. seealso::
+
+   Conference proceedings papers are coordinated by the LSST Project Publication Board.
+   See :doc:`publication-policy` for more information.
+
+.. _presenting-interaction-report:
+
+Interaction report
+==================
+
+As DM team members, we work within an NSF and DOE funded MREFC construction project.
+It's important that we're consistent with the Project when we speak in public and with government officials.
+As you're preparing your presentation:
+
+- `File an interaction report <https://project.lsst.org/interactions>`__. Interaction reports are required when presenting at a public meeting or with government officials.
```

Comment:
I just spoke with Suzanne this morning and I've heard that the "interaction reports" have not been used at all in the modern LSST era. This is likely why the interaction reporting webform is down. I'll remove it.